client/infinite_scroll.js

```diff
@@ -0,0 +1,27 @@
+Template.InfiniteScroll.onCreated(() => {
+  showMoreVisible = (eventName) => {
+    var target = this.$('.infiniteScroller');
+    if (!target.length) return;
+
+    let threshold = $(window).scrollTop() + $(window).height() - target.height();
+
+    if (target.offset().top < threshold) {
+      if (!target.data("visible")) {
+        target.data("visible", true);
+        target.trigger(eventName);
+      }
+    } else {
+      if (target.data("visible")) {
+        target.data("visible", false);
```

Comment:
As discussed you can remove all the `target.data(…)` handling here since you are using the `_.debounce`!